.github/workflows/vulnerability-scan.yml

```diff
@@ -56,4 +56,4 @@ jobs:
           DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
           DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
           LICENSE_KEY: ${{ secrets.DOTCMS_LICENSE }}
-          CUSTOM_STARTER_URL: "https://repo.dotcms.com/artifactory/libs-release-local/com/dotcms/starter/20210429/starter-20210429.zip"
+          CUSTOM_STARTER_URL: "https://repo.dotcms.com/artifactory/libs-release-local/com/dotcms/starter/20210920/starter-20210920.zip"
```

Comment:
Must this file be changed when a new starter is generated? If so, the version should be updated because there is a new starter for 21.12

Reply:
The starter used in the vulnerability scan is way out of date. This is to generate an accurate scan results.

Reply:
so, we don't need to modify this line each time we generate a new starter, do we?

Reply:
I think we do.